pkg/agent/openflow/client.go

```diff
@@ -30,7 +30,12 @@ import (
 	"github.com/vmware-tanzu/antrea/third_party/proxy"
 )
 
-const maxRetryForOFSwitch = 5
+const (
+	maxRetryForOFSwitch = 5
+	// Due to the message size of openflow 1.3 and implementation of Service in Antrea, the maximum Endpoint that Antrea
+	// can support now is 800. If the Endpoints of Service exceed 800, the exceeding Endpoints will be ignored.
```

Comment:
Due to the maximum message size in Openflow 1.3 and the implementation of Services in Antrea, the maximum number of Endpoints that Antrea can support at the moment is 800. If the number of Endpoints for a given Service exceeds 800, extra Endpoints will be dropped.